src/utils/mergeOptions.ts

```diff
@@ -74,6 +74,7 @@ export default function mergeOptions ({
 		cache: getInputOption('cache'),
 		preferConst: getInputOption('preferConst'),
 		experimentalDynamicImport: getInputOption('experimentalDynamicImport'),
+		preserveSymlinks: config.preserveSymlinks,
```

Comment:
`getInputOption` here?

Reply:
(as far as I can tell this would enable `--preserveSymlinks` in CLI but I may be reading this wrong?)

Reply:
I wasn't able to get the CLI option working. Is there some special syntax required?

Reply:
Did you build with `getInputOptions('preserveSymlinks')`?

Reply:
No. I thought you were referring to the code as it exists.

Reply:
Changed to `getInputOption`.